ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1578,6 +1578,35 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
```

Comment:
Use existing `panel_alarm` please.

Reply:
Hi ma'am, 

sometimes i get confused on when I should use specifics e.g. for panel_alarm this is too general as it could mean the panel has alarms going e.g., fire alarm, water leak alarm. So, i use failed_panel_alarm to mean that the actual panel is in error e.g., no feedback from the controller in the panel.